tests/testthat/test-setup.R

```diff
@@ -630,6 +650,11 @@ test_that("`build_setup_pipeline` is working", {
   expect_length(austraits_raw$taxa, 14)
   expect_length(austraits$taxa, 14)
   expect_equal(nrow(austraits$taxa), nrow(austraits_raw$taxa))
+
+  # compare products from two methods, except build_info
+  v <- setdiff(names(austraits), "build_info")
+  expect_equal(tmp_env$austraits[v], austraits[v])
```

Comment:
Do we need a test for the `furrr` method?

Reply:
Ideally, but I've left it out for now. Can you first confirm if it works on your machine?

Reply:
Yes it works